bounties/level-2/stellar-accounts-as-a-service.md

```diff
@@ -0,0 +1,190 @@
+      tags:
+      - auth
+      summary: Create an account.
+      requestBody:
+        description: Credentials to register with.
+        content:
+          application/json:
+            schema:
+              properties:
+                username:
+                  type: string
+                password:
+                  type: string
+        required: true
+      responses:
+        400:
+          description: Invalid body.
+        500:
+          description: Catch all response for other errors aside from the ones listed.
+          content:
+            application/json:
+              schema:
+                properties:
+                  error:
+                    type: string
+                    description: Text explaining the error.
+        
+  /login:
+    post:
+      tags:
+      - auth
+      summary: Get an auth token.
+      requestBody:
+        description: Credentials to login with.
+        content:
+         application/json:
+          schema:
+            properties:
+              username:
+                type: string
+              password:
+                type: string
+        required: true
+      responses:
+        200:
+          description: Successful login.
+          content:
+            application/json:
+              schema:
+                properties:
+                  apikey:
+                    type: string
+        400:
+          description: Invalid Body.
+  /info:
+    get:
+      tags:
+      - info
+      security:
+        - bearerAuth : []
+      summary: Get info about the user's address and XLM balance.
+      responses:
+        200:
+          description: Sucessful operation.
+          content:
+            application/json:
+              schema:
+                properties:
+                  address:
+                    type: string
+                    description: The user's address.
+                  balance:
+                    type: string
+                    description: The user's balance in XLM.
+        401:
+          description: Invalid api key.
+  /pay:
+    post:
+      tags:
+      - pay
+      security:
+       - bearerAuth: []
+      summary: Pay Stellar accounts with XLM.
+      requestBody:
+        required: true
+        description: Destination ddress and amount in XLM
+        content:
+          application/json:
+            schema:
+              properties:
+                destination:
+                  type: string
+                  description: Address to send XLM to.
+                amount:
+                  type: string
+                  description: Amount of XLM to send.
+      responses:
+        200: 
+          description: Successful payment.
+        401:
+          description: Invalid api key.
+        404:
+          description: Dstination address does not exist.
+        400:
+          description: Invalid body
+        500:
+          description: Catch all response for other errors aside from the ones listed.
```

Comment:
I think `insufficient balance` would deserve it's own status. Maybe again `409` (because a conflict between available balance and amount to spend) or [`402`](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#402) - Payment Required (as used by e.g. sipgate)

Reply:
I've changed this in a new commit